app/assets/main/react_components/dashboard/climate_actions/manage_actions/dropdown-kanban/MobileKanbanContainer.jsx

```diff
@@ -11,10 +13,25 @@ const MobileKanbanContainer = ({ children, userActions }) => {
     return actionsAccepted;
   };
 
-  const [showMobileKanban, setShowMobileKanban] = useState(false);
+  const toggleScroll = (bool) => {
+    const scrollable = document.getElementById("scrollable");
+    bool
+      ? (scrollable.className += " overflow-hidden")
+      : (scrollable.className -= " overflow-hidden");
+  };
+
+  useEffect(() => {
+    const toggleMobileKanban = () => setShowMobileKanban(false);
+    const hamburgerMenu = document.getElementById("nav-toggler");
```

Comment:
Maybe use a different name than hamurgerMenu, maybe "mobileDropDownMenu?"

Reply:
I think we would avoid using "mobile" in component names and in variables (except the ones that check if mobile). The components and variables should be independent and when they are rendered should be controlled by the media check. As Simon said once, the component only happens to be used when mobile, it should not depend on it. :)

Reply:
By this I also imply that we should change name on the component itself 🙈